content/source/docs/enterprise/before-installing/network-requirements.html.md

```diff
@@ -9,13 +9,38 @@ The Linux instance that runs Terraform Enterprise needs to allow several kinds o
 
 ## Ingress
 
+### General list of the EXPOSED ports on the instance :
+
 * **22**: To access the instance via SSH from your computer. SSH access to the instance is required for administration and debugging.
 * **80**: To access the Terraform Enterprise application via HTTP. This port redirects to port 443 for HTTPS.
 * **443**: To access the Terraform Enterprise application via HTTPS.
 * **8800**: To access the installer dashboard.
 * **9870-9880 (inclusive)**: For internal communication on the host and its subnet; not publicly accessible.
 * **23000-23100 (inclusive)**: For internal communication on the host and its subnet; not publicly accessible.
 
+### Higher ports and their function in details
+
+* **2003**: Graphite (Carbon) feeding port (monitoring, metrics)
+* **2004**:	Graphite (Carbon) feeding port (monitoring, metrics)
+* **4150-4151, 4160-4161, 4170-4171**:	Replicated NSQD (messaging platform-daemon for internal communication)
+* **5432**:	Internal Postgres
+* **5672**:	RabbitMQ TFE worker coordination
+* **6379**:	Redis (Caching and coordination between web and background workers in the application layer)
+* **7586**:	TFE ingress - pulls in version control systems (VCS) (GitHub, BitBUcket, etc) data and stores via Archivist
```

Comment:
Seems like we made a switch from using parentheses to dashes here. Maybe it's better to use dashes throughout?